localization/english/culture/wc_culture_titles_l_english.yml

```diff
@@ -1,43 +1,344 @@
 ﻿l_english:
+ ####################################################
+ # Special titles
+ ### Places ###
+ city_ahn_qiraj:0 "City"
+ quarter_grim_batol:0 "Quarter"
+ city_grim_batol:0 "City"
 
-####################################################
-#Special titles
+ ### Horde ###
+ warchief_male:0 "Warchief"
+ warchief_female:0 "Warchief"
+
+ ### Scourge ###
+ lich_king_male:0 "Lich King"
+ lich_king_female:0 "Lich Queen"
+
+ ### Night Elves ###
  high_priest_kaldorei_male:0 "$HIGH_PRIEST_MALE_NAME$"
  high_priestess_kaldorei_female:0 "$HIGH_PRIEST_FEMALE_NAME$"
+
+ # Special cultural titles (sorted by culture, then government type).
+ ### Orcs ###
+ # Feudal
+ overlord_male_orc:0 "Overlord"
+ overlord_female_orc:0 "Overlord"
+ high_overlord_male_orc:0 "High Overlord"
+ high_overlord_female_orc:0 "High Overlord"
+ warlord_male_orc:0 "Warlord"
+ warlord_female_orc:0 "Warlord"
+ warchief_male_orc:0 "Warchief"
+ warchief_female_orc:0 "Warchief"
+
+ ### High Elves / Blood Elves ###
+ # Feudal
+ lord_male_highelf:0 "Lord"
+ lady_female_highelf:0 "Lady"
+ lordship_highelf:0 "Lordship"
+ high_king_male_highelf:0 "High King"
+ high_queen_female_highelf:0 "High Queen"
+ high_kingdom_highelf:0 "High Kingdom"
+
+ ### Highbornes / Nightbornes ###
+ # Nightborne specific
+ vicomte_male_nightborne:0 "Vicomte"
```

Comment:
Is there a reason why Nightborne use this title? The leader of Nightborne was Grand Magistrix (https://wowpedia.fandom.com/wiki/Elisande).